src/PKSim.Core/Snapshots/Mappers/ClassificationSnapshotService.cs

```diff
@@ -0,0 +1,110 @@
+using System.Collections.Generic;
```

Comment:
There was a lot of arranging of classifiables and classifications within the project mapper. The service is concerned with how to arrange classifications and classifiables instead.